runs/tutorial_example/chains/multinest_.txt

```diff
@@ -0,0 +1,2 @@
+    0.322344980648892928E-26    0.153469400698917320E+04    0.301938202186512505E+01    0.275005312893483840E+02
```

Comment:
Presumably all these files are for the Jupyter notebooks?

Reply:
Yes, these are example chains, to allow tutorial_five (tutorial_wmap) to show something interesting sooner.